openedx/core/djangoapps/theming/core.py

```diff
@@ -29,20 +29,25 @@ def comprehensive_theme_changes(theme_dir):
         'mako_paths': [],
     }
 
-    templates_dir = theme_dir / "lms" / "templates"
+    if settings.PROJECT_ROOT.endswith("cms"):
```

Comment:
I don't think we actually need to know "that we are in Studio", as much as we need to know "in which root to path to look in for a particular file". In this case, `PROJECT_ROOT`, aka "the directory in which we already look for project resources (`{cms/lms}/{djangoapps/templates/static}/*`)" seems like what we want to be using.

Where I take issue with this is the arbitrary mapping between `cms` and `studio`. Sure, we brand the front-end as `Studio` to users, but this isn't the front-end. Since we providing overrides to layer on top of the traditional project filesystem (`platform:lms/static`, `platform:lms/templates`), it's an oddly confusing indirection to use anything other than `cms`.

I provide an alternative here:
https://github.com/stvstnfrd/edx-platform/commit/82a8d6d8a25a97b26c082521bf5d61c35eef395c

```python
path_project = path_theme / basename(settings.PROJECT_ROOT)
path_templates = path_project / 'templates'
path_static = path_project / 'static'
path_locale = path_project / 'conf' / 'locale'
path_favicon = path_project / 'static' / 'images' / 'favicon.ico'
```

This should ensure a direct, 1-to-1 correspondence.